content/routers/_index.md

```diff
@@ -16,3 +16,19 @@ oc patch \
    --type=merge \
    ingresscontroller/default
 ```
+
+# Moving routers to a specific (infra) node 
+
+Label the desired node with a particular label (infra)
```

Comment:
```suggestion
Label the desired nodes with a particular label (infra) and add a taint to those nodes:
```

Reply:
I am good. Thanks for adding more explanation.

Reply:
Would you mind to add those changes? Thanks!